defaults/main.yml

```diff
@@ -1,9 +1,72 @@
 ---
-# What packages to exclude in yum.conf at install time
-yum_exclude: ""
+#################################################
+# Common settings ###############################
+#################################################
 
 # Time to wait for install to finish (in minutes)
 install_timeout: "30"
 
+# Temp directory for storing provisioning related files
+provision_tempdir: "/tmp"
+
+# Where to store libvirt images and VMs
+libvirt_store: "/var/lib/libvirt"
```

Comment:
If this is used as the value for the image storage, then you should make it `/var/lib/libvirt/images` instead. I believe the `virsh undefine --remove-all-storage` command will fail to properly remove storage if the correct location is not used. The `/var/lib/libvirt/images` location is used by the default storage pool in libvirt. If you want to use `/var/lib/libvirt`, then you have to define that as a target path in a different libvirt storage pool. IMHO it's generally unnecessary to use anything but the default location.